Quicktests fail with Intel oneAPI suite.

We've added quicktests to the github actions workers in #13467.

When running all quicktests on the Intel oneAPI worker, the following errors occur on all 8 quicktests:
```
step.debug: RUN failed. Output:
/home/runner/work/dealii/dealii/tests/quick_tests/step.debug: symbol lookup error: /opt/intel/oneapi/mkl/2022.0.2/lib/intel64/libmkl_gf_lp64.so.2: undefined symbol: mkl_blas_dgemm
tests/quick_tests/CMakeFiles/step.debug.run.dir/build.make:70: recipe for target 'tests/quick_tests/CMakeFiles/step.debug.run' failed
```

Compiling the dummy file as well as all quicktests succeed. These errors only occur at runtime.

--- a/.github/workflows/linux.yml
+++ b/.github/workflows/linux.yml
@@ -238,3 +238,7 @@ jobs:
       run: |
         source /opt/intel/oneapi/setvars.sh
         make -j 2
+    - name: quicktest
+      run: |
+        source /opt/intel/oneapi/setvars.sh
+        make test

